src/pages/Registration.tsx

```diff
@@ -45,7 +45,7 @@ export function getRegistrationFormBlock(): RegistrationFormBlock {
     clientId: Date.now(),
     externalIdentifier: "",
     spatialLocation: -1, // Initialise it as invalid so user has to select something
-    replicateNumber: 0,
+    replicateNumber: "1",
```

Comment:
Can we have it default to "" please?

Reply:
Done. Good point -That will actually remind the user to enter a valid value.